hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthCondition.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.monster.dap
+
+/**
+  *
+  * @param conditionType a name to categorize the condition type by, like "eye", "ear", "cardiac", etc.
+  * @param condition     a specific condition within a condition type, like "blind", "cat", "glauc", "other", etc.
+  * @param congenital    a tuple where the first element is the piece of the target field name that fits in
+  *                      "hs_*_yn", like "cg_eye_blind", and where the second element is the higher level boolean
+  *                      flag name that fits in "hs_*_yn", like "cg_eye_disorders", that the first depends on
```

Comment:
My first assumption is that the order would be flipped (gate / dependent first), but I'll defer to the rest of the team on what's most clear.

Reply:
I don't have a strong preference either way.